bin/transform.py

```diff
@@ -274,5 +331,13 @@ def main():
         metadata_readable.to_csv(RESULTS_PATH, index=False)
         metadata_irida.to_csv(TRANSFORMATION_PATH, index=False)
 
+    elif (args.transformation == CATEGORIZE):
+
+        metadata_readable, metadata_irida = categorize(metadata)
+
+        metadata_readable.to_csv(RESULTS_PATH, index=False)
+        metadata_irida.to_csv(TRANSFORMATION_PATH, index=False)
```

Comment:
This should have `remove_all_NA_columns(metadata_irida)`. `remove_all_NA_columns` and `metadata_irida` specifically. This is so that the Irida file output doesn't have empty columns and deletes pre-existing entries.